packages/explorer/src/EmbeddedExplorer.ts

```diff
@@ -30,8 +31,14 @@ export interface BaseEmbeddableExplorerOptions {
 
   // optional. defaults to `return fetch(url, fetchOptions)`
   handleRequest?: HandleRequest;
-  // defaults to false. If you pass `handleRequest` that will override this.
+  /**
+   * defaults to false. If you pass `handleRequest` that will override this.
+   *
+   * @deprecated Use `initialState.includeCookies` instead
+   */
   includeCookies?: boolean;
+  // optional. defaults to true
```

Comment:
Should we be leaving jsDoc comments so that users can have more insight into what each config option does if they hover over the option?

Comments that start with `//` are pretty much invisible to the user (at least in vscode) unless they go into the package and look at the source.

Reply:
yes lets do docs comments!!

Reply:
Should I change up the other ones too?